index.bs

```diff
@@ -2439,20 +2461,23 @@ prompt string, intended for display on a trusted device on the authenticator.
 : Extension identifier
 :: `txAuthSimple`
 
-: Client argument
+: Client extension input
 :: A single [=UTF-8 encoded=] string prompt.
 
-: Client processing
-:: None, except default forwarding of client argument to authenticator argument.
+: Client extension processing
+:: None, except creating the authenticator extension input from the client extension input.
+
+: Client extension output
+:: Returns the authenticator extension output string as JSON string
```

Comment:
Assuming this is the algorithm to convert the authenticator extension output into the client extension output, you'll want to mention that it [=utf-8 decodes=] the CBOR into a DOMString.